source/educators/migration_wip/1_general_information/resources_for_developers.rst

```diff
@@ -11,7 +11,7 @@ and localizing the code for the edX platform.
 Documentation
 =============
 
-Documentation for developers is available from the `edX Developer Documentation`_ 
+Documentation for developers is available from the `edX Developer Documentation`_
```

Comment:
Out of scope change but I would love this to be 
```suggestion
Documentation for developers is available from the `Open edX Developer Documentation`_
```

😭

Reply:
There are so many of these in the `migration_wip`, but I got this one.